dev-vcs/gitolite/gitolite-3.6.11-r1.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+[[ ${PV} == *9999 ]] && SCM="git-2"
```

Comment:
As `git-2.eclass` is deprecated `EAPI>=6`, I think we better migrate to `git-r3.eclass` even though it is irrelevant in this version.